site2/docs/io-quickstart.md

```diff
@@ -1,136 +1,96 @@
 ---
 id: io-quickstart
-title: "Tutorial: Connecting Pulsar with Apache Cassandra"
-sidebar_label: Getting started
+title: "Tutorial: Connect Pulsar with Database"
+sidebar_label: Get started
```

Comment:
I think there are multiple items in the sidebar using `Getting started`. If you are changing `Getting started` to `Get started`, it might be worth changing them at one PR for consistency.

Reply:
Thanks for reminding, I've fixed all.